repos/system_upgrade/common/libraries/mdraid.py

```diff
@@ -0,0 +1,26 @@
+from leapp.exceptions import StopActorExecution
+from leapp.libraries.stdlib import api, CalledProcessError, run
+
+def is_mdraid_dev(dev):
+    try:
+        result = run(['mdadm', '--query', dev])
+    except CalledProcessError:
+        api.current_logger().warning(
+            'Could not check if device is an md device'
+        )
+        raise StopActorExecution()
+    return '--detail' in result['stdout']
+
+
+def get_component_devices(raid_dev):
+    try:
+        result = run(['mdadm', '--detail', '--verbose', '--brief', raid_dev])
+    except CalledProcessError as err:
+        api.current_logger().warning(
+            'Could not get md array component devices: {}'.format(err)
+        )
+        raise StopActorExecution()
```

Comment:
Would not it be more informative to add exception details to StopActorExecution? Smth like
`raise StopActorExecution('Could not get md array component devices', details={'details': err})` maybe?

Reply:
Here I changed the return type to None, I think that makes more sense.